__tests__/api.test.js

```diff
@@ -69,6 +69,48 @@ describe("/api/articles", () => {
     });
   });
 });
+describe("/api/articles?=", () => {
+  test("should return an array of articles sorted by id in descending order", () => {
+    return request(app)
+      .get("/api/articles?sort_by=article_id&order_by=DESC")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("article_id", { descending: true });
+      });
+  });
+  test("should return an array of articles sorted by votes should default to descending if order_by is not provided", () => {
+    return request(app)
+      .get("/api/articles?sort_by=votes")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("votes", { descending: true });
+      });
+  });
+  test("should return an array of articles in asecending order and default to created_at if sort_by is not provided", () => {
+    return request(app)
+      .get("/api/articles?&order_by=ASC")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("created_at", { ascending: true });
```

Comment:
This is working but not for the reason you think, `ascending` isn't an option as per the jest-sorted docs. However the omission of descending option will default your check to be checking for ascending